docs/getting_started/user-guide.rst

```diff
@@ -1228,6 +1260,63 @@ it could be that a different method of choosing or sorting tags (beyond the defa
 and filter) is needed.
 
 
+Sync Registry
+-------------
+
+A sync is when we take your local filesystem registry, and retrieve updates from the remote defined at 
+``sync_registry`` in your settings.yaml. Since sync will be writing recipes to the filesystem
+it only works if you target a filesystem registry (meaning that the default registry
+as a remote will not work). 
+
+.. note::
+
+   By default, the first filesystem registry found in your settings under the registry list will
+   be used. To provide a one off registry folder (that should exist but does not need to be in your
+   defined list) you can use ``--registry``.
+
+As an example, if we do this without changing the defaults:
+
+.. code-block:: console
+
+    $ shpc sync-registry
+    This command is only supported for a filesystem registry! Add one or use --registry.    
+
+We can then make a dummy directory to support sync. You could also make this directory and add to your settings proper under ``registry``.
+
+.. code-block:: console
+
+    $ mkdir -p ./registry
+    $ shpc sync-registry --registry ./registry
```

Comment:
It should technically clone into a temporary location and then copy all the recipes (based on what you have). The clone you see in the console is into a temporary space. Hmm - do you have a suggestion for how to clone and maintain the executable bit? I wouldn't have even noticed that!

Reply:
"copy all the recipes" ? Not just the recipes but other files too, isn't it ? Otherwise these two scripts couldn't have been modified

Reply:
On the freshest `add/remote-registry` with no local modifications:
```
$ mkdir test_sync
$ shpc config add registry $PWD/test_sync
$ shpc sync-registry --dry-run
(...)
adminer will be added newly.
.github will be added newly.
.github will be added newly.
.github will be added newly.
.github will be added newly.
.github/workflows will be added newly.
.github/workflows will be added newly.
.github/workflows will be added newly.
.github/ISSUE_TEMPLATE will be added newly.
.github/ISSUE_TEMPLATE will be added newly.
.github/ISSUE_TEMPLATE will be added newly.
.github/ISSUE_TEMPLATE will be added newly.
.git will be added newly.
.git will be added newly.
.git will be added newly.
.git will be added newly.
.git will be added newly.
.git will be added newly.
.git/logs will be added newly.
.git/logs/refs/heads will be added newly.
.git/logs/refs/remotes/origin will be added newly.
.git/objects/pack will be added newly.
.git/objects/pack will be added newly.
.git/refs/remotes/origin will be added newly.
.git/refs/heads will be added newly.
.git/info will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
.git/hooks will be added newly.
$ shpc sync-registry
(...)
adminer will be added newly.
.github will be added newly.
.git will be added newly.
```

The perfect solution would be parsing all the container.yaml, extracting the templates they refer too, but also parsing those templates in case they include other files, etc, and also deal with the per-tag yaml file that's been recently introduced. I guess it's technically doable, but the code may be quite complex :)
I would simply suggest to copy everything but the `.git` directory, or everything but the hidden files+directories.